lib/fluent/plugin/in_amqp.rb

```diff
@@ -48,9 +53,20 @@ def configure(conf)
       unless @host && @queue
         raise ConfigError, "'host' and 'queue' must be all specified."
       end
-      @bunny = Bunny.new(:host => @host, :port => @port, :vhost => @vhost,
-                         :pass => @pass, :user => @user, :ssl => @ssl,
-                         :verify_ssl => @verify_ssl, :heartbeat => @heartbeat)
+      if @tls && !(@tls_key && @tls_cert)
```

Comment:
Could you please create method for this section ? Since there are multiple conditions.
Such as below.

```
check_tls_options


...
private
...

def check_tls_options
  if @tls 
    unless @tls_key && @tls_cert
      raise  ConfigError, "'tls_key' and 'tls_cert' must be all specified if tls is enabled."
    end
  end
end
```